torch_geometric/nn/aggr/patch_transformer.py

```diff
@@ -0,0 +1,159 @@
+from typing import Optional
+
+import torch
+from torch import Tensor
+
+from torch_geometric.experimental import disable_dynamic_shapes
+from torch_geometric.nn.aggr import Aggregation
+from torch_geometric.nn.aggr.pt_utils import (
+    FeatEncoder,
+    PositionalEncoding1D,
+    TransformerBlock,
+)
+
+
+class PatchTransformerAggregation(Aggregation):
+    r"""Performs  Heterogeneous Link Encoder embedding
+    and Semantic Patches Fusion
+    from `"Simplifying Temporal Heterogeneous Network for
+    Continuous-Time Link Prediction
+    "<https://dl.acm.org/doi/pdf/10.1145/3583780.3615059>`_paper.
+
+    Args:
+        channels (int): Size of each edge feature
+        use to specify the time embedding dimension of edge timestamps
+        hidden_dim (int, optional): hidden dimension
```

Comment:
nit: we usually use `hidden_channels` in pyg. Also capitalize the first char and adding the dot at last.
```suggestion
        hidden_channels (int, optional): Hidden channels.
```

Reply:
updated in recent commit